<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>firthwindow</class>
 <widget class="QWidget" name="firthwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>40</y>
     <width>395</width>
     <height>194</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>id</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>vocation</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="find">
        <property name="text">
         <string>查找</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>firthwindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
